fluentui_listitem/src/main/res/values/attrs_list_sub_header.xml

```diff
@@ -5,13 +5,13 @@
   -->
 
 <resources>
-    <declare-styleable name="ListSubHeaderView">
-        <attr name="title" />
+    <declare-styleable name="FluentUIListSubHeaderView">
+        <attr name="fluentuiListItemView_title" />
```

Comment:
Similar to fluentuiListItemView_title
Should we add subheader here too?

Reply:
attribute "title" was declared in LineItem declarable-style & resued here. Hence on renaming  "title" to  "fluenuiLineItem_title", it got updated here as well.

Reply:
fluentuiListSubHeaderView_title becuase for ListItemView it is already declared above in attrs_list_item.xml